themes-default/slim/src/components/helpers/select-list.vue

```diff
@@ -92,15 +92,18 @@ export default {
             this.indexCounter += 1;
         },
         addNewItem() {
+            const { newItem, editItems } = this;
             if (this.newItem === '') {
```

Comment:
When comparing use the `const` value, not `this.`

Reply:
little bit down i'm assiging to newItem:
`this.newItem = '';`
Should I still use the const?

Reply:
This this is also correct?:
```
deleteItem(item) {
            const { editItems } = this;
            this.editItems = editItems.filter(e => e !== item);
            this.$refs.newItemInput.focus();
            this.$emit('change', editItems);
        },
```

Reply:
I could go even further and destructure also $refs and $emit?

Reply:
So this would be "correct":
```
deleteItem(item) {
            const { $refs, $emit, editItems } = this;
            this.editItems = editItems.filter(e => e !== item);
            $refs.newItemInput.focus();
            $emit('change', editItems);
        },
```